templates/data/api-gateway-manager-build.yaml

```diff
@@ -62,22 +62,102 @@ templates:
                     - "image_buildpack-golang" # takes latest golang image
                   local:
                     - job_default
+              - jobConfig: # builds an OCI image with api-gateway module from mod-dev branch
+                  name: post-api-gateway-module-build
```

Comment:
Would be good to set slack notifications for this job if it fails

Reply:
In the next commit.